lib/fs.js

```diff
@@ -1921,79 +1921,6 @@ WriteStream.prototype.close = ReadStream.prototype.close;
 // There is no shutdown() for files.
 WriteStream.prototype.destroySoon = WriteStream.prototype.end;
 
-
-// SyncWriteStream is internal. DO NOT USE.
-// Temporary hack for process.stdout and process.stderr when piped to files.
-function SyncWriteStream(fd, options) {
-  Stream.call(this);
-
-  options = options || {};
-
-  this.fd = fd;
-  this.writable = true;
-  this.readable = false;
-  this.autoClose = options.autoClose === undefined ? true : options.autoClose;
-}
-
-util.inherits(SyncWriteStream, Stream);
-
-
-// Export
-Object.defineProperty(fs, 'SyncWriteStream', {
-  configurable: true,
-  writable: true,
-  value: SyncWriteStream
-});
```

Comment:
@Fishrock123
 * `debug` just copied a part of Node.js code, and acknowledges that it's a hack: https://github.com/visionmedia/debug/blob/master/node.js#L137-L142
 * `mock-fs` (and some others) does Node.js version detection and includes a per-major-version copy of the internal `fs` module (presumeably patched). We can ignore that.
 * I suspect some sort of automatic deps packing for browser as the main culpit for most of the other cases. `meteor-please-and-thank-you`, for example, has `pakmanaged.js` which was created by `pakmanager` and actually includes `debug` sources. `jscs` was browserified and also includes `debug` sources.